contents/blog/remote-culture.md

```diff
@@ -0,0 +1,238 @@
+---
+date: 2020-08-28
+title: How to Make a Remote Culture Work
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+![Remote Work Banner](../images/blog/remote-culture/remote-banner.jpg) 
+
+
+<small class="centered-text">Photo by Corinne Kutz</small>
+
+
+<br />
+
+2020 has been a tough year. 
+
+Both in our personal and professional lives, we were all forced to make adjustments. Some small, some big.
+
+For many companies that can afford to do so, 2020 has become the year of "work from home". 
+
+A quote from an acquaintance of mine, an executive at a large European firm, sums it up quite well:
+
+<blockquote class="blog-quote">
```

Comment:
The style of your openings is good, but this is too generic. Everyone knows this already - I'd not read an article like this.

You could be more like:

"Many companies are having to adjust to remote work, but what about the ones that started this way from scratch?

PostHog has a team of 12, 10 of whom have never met each other in real life.

What's it like building a company for remote from the first day?"

Reply:
Agreed, great!